src/isIsogram.js

```diff
@@ -20,3 +15,14 @@ function isIsogram(word) {
 }
 
 module.exports = { isIsogram };
+
+
+function maxValue(array) {
+  return Math.max(...arr);
+}
+
+function isPalindrome(str) {
+  const reversedString = str.split('').reverse().join('');
+
+  return str === reversedString;
+}
```

Comment:
The function `isPalindrome` is also not related to `isIsogram` and it's not exported. If it's not needed for this file, consider removing it. If it's needed elsewhere, consider moving it to a more appropriate location.